packages/website/pages/files.js

```diff
@@ -142,6 +142,7 @@ const UploadItem = ({ upload, index, toggle, selectedFiles, showCopiedMessage })
       setError('')
     } catch(e) {
       console.error(e);
+      // @ts-ignore Catch clause variable type annotation must be 'any' or 'unknown' if specified.ts(1196)
```

Comment:
This is a side effect of updating the ts compiler dep, which was making cloudflare pages CI job to fail. We seem to assume it will always be an Error we know about, and probably this is a safe assumption, but I am flagging it.